server/store/sqlstore/migrations.go

```diff
@@ -0,0 +1,15 @@
+package sqlstore
+
+import (
+	sq "github.com/Masterminds/squirrel"
+)
+
+func (s *SQLStore) runMigrationRemoteID(remoteID string) error {
+	_, err := s.getQueryBuilder().Update("Users").Set("RemoteID", remoteID).Where(sq.And{
+		sq.NotEq{"RemoteID": nil},
+		sq.NotEq{"RemoteID": ""},
```

Comment:
To confirm, all users we've ever created have had a non-nil, non-empty `RemoteID`?

Reply:
All the synthetic users, by definition, are the ones that have `RemoteID` different that nil/"", so. Not "all the users" that we have created, the bot user, and any promoted user has no `RemoteID`, but that is the expected behavior.

Reply:
Awesome, thanks for confirming! (Mostly was just thinking back to early deployments with any customers and making sure there wasn't a time when `RemoteID` was left `nil`.)